api/src/test/java/org/apache/iceberg/expressions/TestExpressionUtil.java

```diff
@@ -18,21 +18,33 @@
  */
 package org.apache.iceberg.expressions;
 
+import static org.apache.iceberg.expressions.Expressions.equal;
```

Comment:
You are mixing Expressions.equal and static equal() call. Can you use Expressions.equal() for all of them?